source/documentation/datatypes/datatypes.md

```diff
@@ -0,0 +1,85 @@
+## Boolean
+
+A datatype which can have one of two states, `True` and `False`. 
+
+## CURIE
+
+Inspired by the [CURIE Syntax 1.0](https://www.w3.org/TR/curie/), the `CURIE`
+(compact URL) datatype provides a mechanism [to link
+registers](/linked_registers).
+
+CURIEs are of the form `{prefix}:{key}`, where `{prefix}` is mapped to a base
+URL `https://{name-of-register}.register.gov.uk/records/`.
```

Comment:
In here you are relying on the string representation but you don't let users know that.